Source/WebCore/page/IntersectionObserver.idl

```diff
@@ -49,5 +49,4 @@
 dictionary IntersectionObserverInit {
     (Element or Document)? root = null;
     DOMString rootMargin = "0px";
-    (double or sequence<double>) threshold = 0.0;
```

Comment:
Why are we moving this? That doesn't seem right.

Reply:
I removed it because it was referencing `squence<double`.

Reply:
I don't think we should remove this sequence. It's still in the spec as well.

Reply:
Undid it - Done in next commit `9298eb6`.